gumpy/difference.py

```diff
@@ -239,6 +240,16 @@ def get_gene_pos(
             if self.genome2.genes[gene]["reverse_complement"]:
                 nc_num = nc_num - dels + 1
 
+                # Edge case of deletion starting in revcomp gene and extending
+                # past gene start, so return None
+                if (
+                    self.genome2.stacked_nucleotide_number[
+                        self.genome2.stacked_gene_name == gene
+                    ][-1]
+                    > nc_num
+                ):
+                    return None
```

Comment:
Is it easy to add a test case for this just to make sure there aren't any off by one errors, or something like that?

Reply:
It's not particularly easy to add a test case for this - it'd either have to be a new dummy genome or a contrived TB test case (slow). However, it shouldn't be off by one as `>` should be checking that the calculated `nc_num` is less than the minimum gene position